web/src/utils/storybook.ts

```diff
@@ -60,6 +62,22 @@ const getPastValues = () => {
         rh_tgl_2_median: rh,
         rh_tgl_2_90th: rh - 3
       })
+      _pastRegionalModelValues.push({
+        datetime,
+        temperature: temp + (Math.random() - 0.7) * 7,
```

Comment:
You could use a sine wave here to get a more natural curve going.

Reply:
`temperature` is based off `temp`, which uses the sine curve.